packages/esm-patient-forms-app/src/index.ts

```diff
@@ -38,6 +38,8 @@ function setupOpenMRS() {
   defineConfigSchema(moduleName, {});
 
   return {
+    lifecycle: getAsyncLifecycle(() => import('./forms/forms-see-all.component'), options),
+    activate: 'forms-see-all.component',
```

Comment:
This is called "route" and I think the route should be more adequate for this scenario.

Reply:
I could have missed the topic on "route" in the 3.0 docs. It would be helpful if you could point me to the OpenMRS docs about "route". Thank you.

Reply:
`activate` is the old name, `route` is the new name. (See https://openmrs.github.io/openmrs-esm-core/#/main/creating_a_microfrontend - there is `load` and `route`, while you use `lifecycle` and `activate` -- these two are outdated and should be replaced with the new names)

Independent of the property name the exact route should be adjusted, too. As written, a route called "forms-see.all.component` is not good. A route should be something like `all-forms` or `forms/all` etc.

Reply:
Have made the changes accordingly, Thank you.